Formula/gmp.rb

```diff
@@ -16,12 +16,13 @@ class Gmp < Formula
 
   def install
     ENV.cxx11 if build.cxx11?
-    args = ["--prefix=#{prefix}", "--enable-cxx"]
 
-    # https://github.com/Homebrew/homebrew/issues/20693
-    args << "--disable-assembly" if build.bottle?
+    # XNU release major version
+    osmajor = MacOS.version.to_s[3..-1].to_i + 4
```

Comment:
What does the output end up being? Should rather shell out for this if we can; fairly opaque as-is (not your fault)